python/cugraph/structure/graph.py

```diff
@@ -445,6 +456,10 @@ def from_dask_cudf_edgelist(self, input_ddf, source='source',
         renumber : bool
             If source and destination indices are not in range 0 to V where V
             is number of vertices, renumber argument should be True.
+        store_transposed : bool
```

Comment:
Do we really need to re-pass flag here in addition to the graph constructor? 
It seems this could lead to undefined behaviour if we pass Ture in the former and False in the later and vice versa.

Reply:
It isn't in the graph constructor, it is in the `from_cudf_edgelist` and `from_dask_cudf_edgelist` calls.  If we were to call one and then the other that would raise an exception.